### DISCUSSION TITLE
New SDK and frameworks (like Angular)

### DISCUSSION THREAD

**Discussion by bogacg** [firebase/firebase-js-sdk#4387] [General]
I love the new approach and immediately I thought: "If this is modular and lightweight, then maybe libraries like `angularfire` becomes obsolete?"

I'm going to try it but please share if there is already a starter/demo for such use case.

**Comment by davideast** [comment]
Hey @bogacg! That is definitely not the case. AngularFire provides many, many things for developers. Such as

- Dependency Injection
- Angular's Module system
- Observable support
- And most importantly, Zone.js coordination (this is a hidden but but super important detail)

The same goes for many other libraries like RxFire and ReactFire. It's important to provide idiomatic extensions such as Observables for RxFire and Hooks, Suspense, and Context for ReactFire.
